### DISCUSSION TITLE
Implement Duplication Check for Prompt Versions in Langfuse

### DISCUSSION THREAD

**Discussion by sapipoZZZ** [langfuse/langfuse#2390] [Ideas]
### Describe the feature or potential improvement

I need to implement a duplication check in the Prompt version of Langfuse. If the submitted Prompt this time is exactly the same as the one in the current latest version, do not create two identical versions.

[v2.47.5]

### Additional information

![image](https://github.com/langfuse/langfuse/assets/19617283/eaf3dfa2-b299-4912-9a01-6e279225e71b)
![image](https://github.com/langfuse/langfuse/assets/19617283/97756d1c-0f5b-4539-b0b0-0bd6dec37c65)

**Comment by marcklingen** [comment]
How do you create the duplicated versions?
